droid-core/src/test/java/uk/gov/nationalarchives/droid/core/signature/compiler/ByteSequenceSerializerTest.java

```diff
@@ -103,12 +465,11 @@ private void testRenderSignatures(String fileName, SignatureType type) throws Ex
             if (sequence.trim().length() > 0) { //ignore blank lines in the test input files.
```

Comment:
defence against possible empty lines 👍🏼 
Should we also defend against possible blank space characters (i.e. would it be better to trim the sequence before passing it into `compileExpression`?)

Reply:
I put this in because I occasionally left a blank line or spaces in the test files.  This just ensures we're always testing an actual expression - a line with something in it.

Reply:
Thanks 👍🏼